chain/transaction.go

```diff
@@ -31,68 +31,71 @@ var (
 type Transaction struct {
 	Base *Base `json:"base"`
 
-	Actions []Action `json:"actions"`
-	Auth    Auth     `json:"auth"`
+	Actions Actions `json:"actions"`
+	Auth    Auth    `json:"auth"`
 
-	digest    []byte
+	preimage  []byte
 	bytes     []byte
 	size      int
 	id        ids.ID
 	stateKeys state.Keys
 }
 
-func NewTx(base *Base, actions []Action) *Transaction {
+func NewTx(base *Base, actions Actions) *Transaction {
 	return &Transaction{
 		Base:    base,
 		Actions: actions,
 	}
 }
 
-func (t *Transaction) Digest() ([]byte, error) {
-	if len(t.digest) > 0 {
-		return t.digest, nil
+// Digest returns the byte slice representation of the unsigned tx
```

Comment:
nit: update the comment to match the function name

Reply:
done